src/StatusQ/Components/StatusLetterIdenticon.qml

```diff
@@ -0,0 +1,26 @@
+import QtQuick 2.13
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+Rectangle {
+    id: statusLetterIdenticon
+
+    property string name
+    property int letterSize: 21
+
+    color: Theme.palette.miscColor5
+    width: 40
+    height: 40
+    radius: width / 2
+
+    StatusBaseText {
+        text: (statusLetterIdenticon.name.charAt(0) == "#" ? statusLetterIdenticon.name.charAt(1) : statusLetterIdenticon.name.charAt(0)).toUpperCase()
+        font.weight: Font.Bold
```

Comment:
The `font.weight` isn't applying here because we aren't loading all font weights of our custom fonts at the moment.

This is fixed with #31 but no changes in this PR are required then.